public/app/core/components/NavBar/Next/NavBarMenu.tsx

```diff
@@ -104,6 +113,15 @@ const getStyles = (theme: GrafanaTheme2) => ({
     flexDirection: 'column',
     overflow: 'auto',
   }),
+  mobileHeader: css({
+    borderBottom: `1px solid ${theme.colors.border.weak}`,
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: `${theme.spacing(1)} ${theme.spacing(2)} ${theme.spacing(2)}`,
```

Comment:
I could be wrong but [you should be able to](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/createSpacing.test.ts#L10) pass multiple args to `theme.spacing`.

```suggestion
    padding: `${theme.spacing(1, 2, 2)}`,
```

Reply:
GAMECHANGER 🚀